docker/mariadb/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1.4
+FROM mariadb:10.11.4
+
+WORKDIR /docker-entrypoint-initdb.d
+
+RUN apt-get update && \
+    apt-get install -y wget && \
+    wget https://redmine.seichi.click/attachments/download/995/gachadata.sql && \
```

Comment:
[`ADD` コマンド](https://docs.docker.com/engine/reference/builder/#add)でいい説あります

> `ADD` obeys the following rules:
> - [...]
> - [...]
> - If `<src>` is a URL and `<dest>` does end with a trailing slash, then the filename is inferred from the URL and the file is downloaded to `<dest>/<filename>`. For instance, `ADD http://example.com/foobar /` would create the file `/foobar`. The URL must have a nontrivial path so that an appropriate filename can be discovered in this case (`http://example.com` will not work).